README.md

```diff
@@ -217,18 +217,20 @@ The payload MUST describe the authorization claims, who is involved, and its val
 | `exp` | `Number`   | Expiration UTC Unix Timestamp (valid until)      | Yes      |
 | `nnc` | `String`   | Nonce                                            | No       |
 | `fct` | `Json[]`   | Facts (asserted, signed data)                    | No       |
-| `prf` | `String[]` | Proof of delegation (witnesses are nested UCANs) | Yes      |
+| `prf` | `String[]` | Proof of delegation (nested UCANs)               | Yes      |
 | `att` | `Json[]`   | Attenuations                                     | Yes      |
 
 ### 3.2.1 Principals
 
 The `iss` and `aud` fields describe the token's principals. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with the private key associated with the DID in the `iss` field. Implementations MUST include the [`did:key` method](https://w3c-ccg.github.io/did-method-key/), and MAY be augmented with [additional DID methods](https://www.w3.org/TR/did-core/).
 
+The underlying key types RSA, ECDSA, and EdDSA MUST be supported. Use of ECDSA is supported but [RECOMMENDED that another key type be preferred](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Security).
```

Comment:
```suggestion
The underlying key types and signature algorithms RSA, ECDSA, and EdDSA MUST be supported. Use of ECDSA is supported but [RECOMMENDED that another key type be preferred](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Security).
```

Key types are one part of the story. Another one is how you sign with them.

Reply:
Would that not be covered by the `alg` field in the JWT header?

Reply:
Hmm, true. Please, disregard the suggestion then.